messagebus/src/vespa/messagebus/network/rpcsend.cpp

```diff
@@ -147,6 +147,11 @@ RPCSend::send(RoutingNode &recipient, const vespalib::Version &version,
 void
 RPCSend::RequestDone(FRT_RPCRequest *req)
 {
+    _net->getExecutor().execute(vespalib::makeLambdaTask([this, req]() { doRequestDone(req);}));
```

Comment:
need to handle rejected tasks

Reply:
Asserting that they are not rejected.

Reply:
The assert was added to the other place where tasks might be rejected